codepipeline_deploy.sh

```diff
@@ -12,6 +11,22 @@ if [ -z "${PIPELINE_TIMEOUT}" ]; then
     PIPELINE_TIMEOUT=1800
 fi
 
+# Get the HEAD commit ID for version.json in master branch if exists
+MASTER_COMMIT_ID=`aws codecommit get-file --repository-name AppManifest-"$SA_NAME"-"$ROS_DISTRO"-gazebo"$GAZEBO_VERSION" --file-path version.json`
+if [ $? -ne 0 ]; then
+    echo version.json does not yet exist in CC repository named "$CC_REPO_NAME"
+else
+    export MASTER_COMMIT_ID=`echo $MASTER_COMMIT_ID | jq -r '.commitId'`
+fi
+
+# Upload version.json to CodeCommit
+if [ -z "${MASTER_COMMIT_ID}" ]; then
+    # version.json file being uploaded the first time
+    aws codecommit put-file --repository-name AppManifest-"$SA_NAME"-"$ROS_DISTRO"-gazebo"$GAZEBO_VERSION" --branch-name master --file-content file://"$TRAVIS_BUILD_DIR/shared/version.json" --file-path version.json --commit-message "$TRAVIS_COMMIT_MESSAGE" --name "$GH_USER_NAME" --email "$GH_USER_EMAIL" --file-mode normal
```

Comment:
Consolidate these (25, 27) into a single line because they're mostly identical. Just use a helper variable like `$EXTRA_PARAMS`, which would be set to `--parent-commit-id "$MASTER_COMMIT_ID"` when you set MASTER_COMMIT_ID above.

Reply:
I have remove use of codecommit to upload the version.json file. please take a look.

Reply:
> Do you have a success build for this? I usually test this by creating a branch for a sample app that uses the PR branch for travis-script instead of master, and run that on travis

https://travis-ci.org/aws-robotics/aws-robomaker-sample-application-helloworld/builds/513230702